O/open62541/build_tarballs.jl

```diff
@@ -44,7 +44,9 @@ products = [
 ]
 
 # Dependencies that must be installed before this package can be built
-dependencies = Dependency[
+dependencies = [
+    Dependency("OpenSSL_jll"; compat="3.0.8"),
+    HostBuildDependency("OpenSSL_jll"; compat="3.0.8")
```

Comment:
Why `HostBuildDependency`?  That looks completely wrong.
```suggestion
    Dependency("OpenSSL_jll"; compat="3.0.8")
```

Reply:
because I didn't RTFM enough and was just trying, sorry.